src/main/com/smithsmodding/armory/common/logic/initialization/CommonSystemInitializer.java

```diff
@@ -171,7 +111,9 @@ private static void tryRemoveRecipeFromGame(@Nonnull IRecipe recipe, @Nonnull It
                         try {
                             iterator.remove();
                             return;
-                        } catch (IllegalStateException ex) {
+                        }
+                        catch (Exception ex)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)